install-tests/react-native/patches/App.js

```diff
@@ -33,13 +37,29 @@ const App = () => {
         realm.create("Person", { name: "Charlie" });
       });
     }
+
+    function testExceptions() {
+      try {
+        // Historically, we've seen crashes when throwing JS exceptions from functions called by C++
+        // This is often caused by an NDK mismatch between prebuild shared-object files.
+        realm.write(() => {
+          throw new Error("!");
+        });
+      } catch (err) {
+        return err.message;
+      }
+    }
+
+    const suffix = testExceptions();
+
     // Read the persons out of the database again
     const message =
       "Persons are " +
       realm
         .objects("Person")
         .map((p) => p.name)
-        .join(", ");
+        .join(", ") +
+      suffix;
```

Comment:
If i'm understanding this right, this will output a "!" when successful?  Is that meaningful?

Reply:
> Is that meaningful?

Not really 🙂 I just need some string to round-trip and expect.